src/components/common/icons/ChevronIcon.tsx

```diff
@@ -1,9 +1,26 @@
+import { css } from '@emotion/react';
+
 import Svg, { SvgProps } from '~/components/common/Svg';
 
-export function ChevronIcon({ ...props }: SvgProps) {
+export interface Props extends SvgProps {
+  direction?: 'up' | 'right' | 'down' | 'left';
+}
```

Comment:
```suggestion
export interface ChevronIconProps extends SvgProps {
  direction?: 'up' | 'right' | 'down' | 'left';
}
```

위에서 언급되었던 비슷한 이유로 해당 Props도 이름을 바꿔주는 것이 좋을 것 같다는 생각이에요~!

혹은 혜성님이 사용하시는 방식으로 Props는 export 하지 않고, 사용 시에 `ComponentProps<typeof Component>`처럼 사용하는 것도 방법이 될 수 있을 것 같아요